spec/rails_helper.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+require 'shoulda/matchers'
+require 'support/shoulda'
+require 'support/request_helpers'
+require 'webmock/rspec'
+require 'simplecov'
+
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+
+require 'rspec/rails'
+
+ActiveRecord::Migration.maintain_test_schema!
+
+Dir[Rails.root.join('spec/support/**/*.rb')].each(&method(:require))
+
+FactoryBot.definition_file_paths << File.expand_path('../support/factories', __FILE__)
+
+WebMock.disable_net_connect!(allow_localhost: true)
+
+# Requires supporting ruby files with custom matchers and macros, etc, in
+# spec/support/ and its subdirectories.
+
+RSpec.configure do |config|
+  config.use_transactional_fixtures = true
+  config.infer_spec_type_from_file_location!
+  config.filter_rails_from_backtrace!
+
+  config.before :suite do
+    DatabaseCleaner.clean_with(:truncation)
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before :each do
+    DatabaseCleaner.start
+  end
+
+  config.after :each do
+    DatabaseCleaner.clean
+  end
+
+  config.after :all do
+    FileUtils.rm_rf Dir["#{Rails.root}/spec/test_files/"]
```

Comment:
:scream: That's not a raw Ruby script, don't work with file system in such explicit ways

Remove this block